index.html

```diff
@@ -371,7 +371,7 @@ <h3 class="about__connect fz-16 fw-500 lh-24 clr-p-400">
                         </li>
                         <li class="about__icons__li">
                             <a href="#" class="about__icons__link link"><img
-                                    src="images/m.png" alt="M icon" /></a>
+                                    src="images/m.png" alt="Medium icon" /></a>
```

Comment:
- [ ] Please remove the text "icon", even don't use the words like `button`, `icon`, or `image of` in your **alternative-text** because by default when a person uses screen readers, the screen reader automatically provides the context. Kindly use "Medium Profile" because it will assist the person to know whether he/she is interested to look into your `profile` or not.